app/backend/src/couchers/models.py

```diff
@@ -101,7 +111,7 @@ class User(Base):
     # profile color
     color = Column(String, nullable=False, default="#643073")
     avatar_filename = Column(String, nullable=True)
-    # TODO: array types once we go postgres
+    # deprecated
     languages = Column(String, nullable=True)
```

Comment:
You can remove this. We've made no commitment to keep languages in the database through alpha, so we can just nuke this and people can re-add their languages. It's easier than writing migrations to match them up.